server/utils/raiseAnalyticsEvent.ts

```diff
@@ -0,0 +1,19 @@
+import ua from 'universal-analytics'
+import config from '../config'
+
+// @ts-ignore
+export const raiseAnalyticsEvent = (category, action, label, value?) => {
+  if (!config.googleAnalytics.ga4MeasurementId) return Promise.resolve()
+  const ga = ua(config.googleAnalytics.ga4MeasurementId)
+  const data = {
+    ec: category,
+    ea: action,
+    el: label,
+    ev: value,
+  }
+  return ga.event(data).send()
+}
```

Comment:
i think this whole thing should be removed (and references to it) because we do not do this for any other server-side actions and it won't work anyway as it's trying to use google universal analytics rather than google analytics 4